exporters/otlp/all/build.gradle.kts

```diff
@@ -41,6 +39,45 @@ val testJavaVersion: String? by project
 
 testing {
   suites {
+    listOf(
+      "LATEST",
```

Comment:
does the latest version still get tested outside of this new suite also?

Reply:
Indirectly via [NoGrpcJavaOtlpIntegrationTest](https://github.com/open-telemetry/opentelemetry-java/blob/main/integration-tests/otlp/src/test/java/io/opentelemetry/integrationtest/NoGrpcJavaOtlpIntegrationTest.java).

But the code for testing the OkHttp based sender was previously in the default `test` source set. Those test classes have been relocated to `testDefaultSender` in this PR, which runs as part of these new test suites. 

The latest will still always run locally (not just on CI), because of the line further down:
```
enabled = it.equals("LATEST") || "true".equals(System.getenv("CI"))
```

Reply:
> Those test classes have been relocated to testDefaultSender in this PR, which runs as part of these new test suites.

ah, that's what I missed, thx!